src/interface/src/environments/environment.ts

```diff
@@ -5,7 +5,8 @@
 export const environment = {
   production: false,
   backend_endpoint: 'http://localhost:8000/planscape-backend',
-  google_analytics_id: '' // Replace with actual ID.
+  google_analytics_id: '',  // Replace with actual ID.
```

Comment:
can you add the same change to environment.prod.ts?

Reply:
I made these changes directly in github because the force commit flag wasn't working for me for some reason, so I couldn't make both changes in the same branch :( . That change is in pr#790